worker/src/mirrors_qa_worker/manager/worker.py

```diff
@@ -0,0 +1,27 @@
+import os
+import time
+
+from mirrors_qa_worker import logger
+
+
+class WorkerManager:
+    """Manager responsible for creating tasks"""
+
+    # number of seconds between each poll
+    sleep_interval = int(os.getenv("SLEEP_INTERVAL", 180))  # noqa
```

Comment:
I know this is a stub but we'd still be using something similar to the Settings in the backend: a central place where we fetch and validate inputs (from environment variables).
Makes it very easy to discover what can be changed